internal/cmd/asyncapi/command_import.go

```diff
@@ -41,7 +41,8 @@ type flagsImport struct {
 }
 
 type kafkaBinding struct {
-	XConfigs map[string]string `yaml:"x-configs"`
+	XPartitions int32             `yaml:"x-partitions"`
+	XConfigs    map[string]string `yaml:"x-configs"`
```

Comment:
Should these also be `omitempty`?

Reply:
No point in making these omitempty because we are not outputting the yaml here. We are reading from a yaml file.